libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -68,29 +68,56 @@ const decorateLinkGroup = (elem, index) => {
   return linkGroup;
 };
 
-// Decorate a list of simple links
-const decorateLinkList = ({ list, className = 'feds-navLink', includeCta = true } = {}) => {
-  list.querySelectorAll('a').forEach((link, index) => {
+const decorateElements = ({ elem, className = 'feds-navLink', parseCtas = true, index = { position: 0 } } = {}) => {
```

Comment:
Why use `index.position` vs just `index`?  Doesn't look like the index obj has any other properties

Reply:
Because the index needs to be synced across blocks used as part of a menu, I pass the object wherever it's needed, thus all blocks can update its `index` property when they need to and all of the other blocks get the update via the object. It's sort of a shared context without declaring a global variable (which I've also tried, but since there is so much async logic, it leads to inconsistent values).